crates/dkg-core/src/primitives/joint_feldman.rs

```diff
@@ -69,7 +69,7 @@ impl<C: Curve> DKG<C> {
 
     /// Creates a new DKG instance from the provided private key, group and RNG.
     ///
-    /// The private key must be part of the group, otherwise this will return an error.
+    /// The public key must be part of the group, otherwise this will return an error.
```

Comment:
Change the comment on the new function as well? Looks like it has the same error